lib/utils/testLauncherHelper.js

```diff
@@ -422,8 +423,7 @@ async function prepareTestPackExecution(ownArgv) {
 	const deviceCount = devices.length;
 
 	if (!deviceCount) {
```

Comment:
I wonder whether we still want to send closeSession request ?

Reply:
It will be done in finally block in SuitestLancher

Reply:
@nivanis91 Could you please double-check this one? If it's OK - we'll merge the PR.

Reply:
Yep, it's done in finally block, as @JeKachan  says.